src/deterministic.js

```diff
@@ -12,7 +12,7 @@ Math.random = () => {
 var TIME = 10000;
 Date.now = () => TIME++;
 if (typeof performance == 'object') performance.now = Date.now;
-if (ENVIRONMENT_IS_NODE) process['hrtime'] = Date.now;
+if (ENVIRONMENT_IS_NODE) process.hrtime = Date.now;
```

Comment:
Without the quoting, isn't there a risk that closure minifies `process.hrtime` to say `process.pid`, which already exists and would conflict? I think to be safe we need to quote this.

Reply:
(or put `hrtime` in the externs)

Reply:
`process.hrtime` is a known function and should not get minified by closure:  https://nodejs.org/api/process.html#processhrtimetime

Reply:
Hmm.. it does seem to be not present in any of the externs files though, though, which is odd because I would have though that would make it get minified and then emscripten_get_now would crash... which doesn't happy.  So somehow its not being minified.

Reply:
Looks like it get defined in `third_party/closure-compiler/node-externs/process.js`